backend/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -6162,7 +6162,7 @@ databaseChangeLog:
                   type: text
                   remarks: The SNOMED code for this specimen
                   constraints:
-                    unique: true
+                    unique: false
```

Comment:
I believe updating a previous changeset like this will break things.  Solution would be to make this update in a subsequent changeset

Reply:
I did add a new changeset to fix this, so you are correct I need to revert this.